app/metrics.yaml

```diff
@@ -5809,3 +5809,156 @@ recent_tabs:
     notification_emails:
       - android-probes@mozilla.com
     expires: "2022-06-23"
+credit_cards:
+  saved:
+    type: counter
+    description: |
+      A counter of the number of credit cards that have been saved
+      manually by the user.
+    send_in_pings:
+      - metrics
```

Comment:
Note that this property is not required, unless you specify a custom ping. This applies to the other definitions as well.

Reply:
Understood.  I'll remove the `send_in_pings` property.  Thanks!